client/src/test/component/ProfileForm.spec.js

```diff
@@ -0,0 +1,35 @@
+import expect from 'expect';
+import React from 'react';
+import { mount, shallow } from 'enzyme';
```

Comment:
@andela-oowonikoko This is a valid comment. Don't import `shallow` or `mount` if you are not using it.